torch/_inductor/pattern_matcher.py

```diff
@@ -354,8 +354,17 @@ class _TargetArgsExpr(_TargetExpr):
     Base class for filtering match by node.{target,args,kwargs}
     """
 
-    def __init__(self, fns, *args, _users=1, **kwargs):
+    def __init__(self, fns, *args, _users=1, is_optional=None, **kwargs):
         super().__init__(fns, _users)
+
+        if is_optional is None:
+            maybe_optional_ops = (aten.mul.Tensor, aten.add.Tensor, aten.sub.Tensor, aten.div.Tensor)
+            if len(self.fns) == 1 and self.fns[0] in maybe_optional_ops:
```

Comment:
To expand on my reasoning, I was thinking about adding a pattern `a.mul(b).add(c) -> fma(a, b, c)`. With this PR I think that pattern would match literally everything since both `CallFunction` nodes would be optional.

Reply:
> Can we pass is_optional=True explicitly in the pattern definition?

Yes, it can be possible as well. Discussing some time ago with Mario (cc @lezcano ) he suggested to set default to None and resolve it to true for this list of ops.

> Could we just add more pattern definitions for quantization? 

yes, this can be an option but would require a bit more code when generating patterns.